### PR TITLE
gpgme: Add gpgme1.0

### PR DESCRIPTION
**Pull request by masami256**
At first, actual gpgme package name in debian buster is gpgme1.0.

Debian disable python3 and python2 as valid binding language name by
0003-python-Auto-check-for-all-installed-pyth.patch but poky uses
python2 and python3 as lang name. So revert that patch to support
python[23].

### CODE REVIEW

File: recipes-debian/gpgme/gpgme_debian.bb [hunk 1]
```diff
@@ -0,0 +1,90 @@
+SUMMARY = "High-level GnuPG encryption/signing API"
```
Comment (hiraku-wfs) [inline recipes-debian/gpgme/gpgme_debian.bb:1]: Could you add description header if you refered to poky's recipe?

File: recipes-debian/gpgme/gpgme_debian.bb [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+# Default in configure.ac: "cl cpp python qt"
+# Supported: "cl cpp python python2 python3 qt"
+# python says 'search and find python2 or python3'
+
+# Building the C++ bindings for native requires a C++ compiler with C++11
+# support. Since these bindings are currently not needed, we can disable them.
+DEFAULT_LANGUAGES = ""
+DEFAULT_LANGUAGES_class-target = "cpp"
+LANGUAGES ?= "${DEFAULT_LANGUAGES}"
+#LANGUAGES .= "${@bb.utils.contains('PACKAGECONFIG', 'python2', ' python2', '', d)}"
```
Comment (hiraku-wfs) [inline recipes-debian/gpgme/gpgme_debian.bb:52]: Why is this line commented out?

Comment (masami256) [reply recipes-debian/gpgme/gpgme_debian.bb:52]: ah, sorry. It's for debug. It should be enabled. I'll remove comment.

File: recipes-debian/gpgme/gpgme_debian.bb (hunk 1 above)
Comment (masami256) [reply recipes-debian/gpgme/gpgme_debian.bb:1]: I see

**Comment by masami256** [comment]
I've created version 2 of pull request at #46

**Pull request closed without merging**